__tests__/commands/install/integration.js

```diff
@@ -491,6 +491,19 @@ test.concurrent('install file: dedupe dependencies 2', (): Promise<void> => {
   });
 });
 
+// When local packages are installed from a repo with a lockfile, the multiple packages
+// unpacking in the same location warning should not occur
+test.concurrent('install file: dedupe dependencies 3', (): Promise<void> => {
+  return runInstall({}, 'install-file-dedupe-dependencies-3', (config, reporter, install, getStdout) => {
+    const stdout = getStdout();
+    // Need to check if message is logged, but don't need to check for any specific parameters
+    // so splitting on undefined and testing if all message parts are in stdout
```

Comment:
I feel like we can simply use `expect(getStdout()).toMatchSnapshot()` here with a comment similar to what you already put there. Thoughts?